website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md

```diff
@@ -2,159 +2,59 @@
-
-#### Fetch Single File
-
-  <VideoEmbed host="youtube" videoId="dVZEd8p0Y2c" /> 
-
-* Click the + icon next to the **queries/js** and choose your **S3 datasource.**
-* Rename the query.
-* From the Commands drop-down, Select the method **Read file.**
-
-You can pass the below parameters to **Read a file.**
-
-* **Bucket Name:** Name of the bucket where the image is stored.
-* **File Path**: Path of the image you want to fetch. ex. images/name.
-* **File Data Type**: You should select base64 to display the image.
-
-Once you have added all the required parameters:
-
-* Drag and drop the [Image widget](/reference/widgets/image) onto the canvas.
-* In the Image property pane, add:
-
-```
-{{<your_query_name>.data.fileData}}
-```
-
-#### Fetch all files
-
-  <VideoEmbed host="youtube" videoId="UzV5LZ0kvDQ" /> 
-
-* Click the + icon next to the **queries/js** and choose your **S3 datasource.**
-* Rename the query.
-* From the Commands drop-down, Select the method **List files** in the bucket.
-* Add the **bucket name.**
-* Now, run the query.
-
-Now, open the query window and select the table option on the right-side property pane. It would automatically add a table widget to your canvas.
-
-:::info
-Bind the query’s response to the Table using JavaScript in the Table Data Property **`{{list_files.data}}`**.
-:::
-
-Now your table should list all the files present in your S3 bucket.
-
-You can use an image widget to display images listed in the table widget. You can follow this [guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more.
-
-#### Download Files
-
-* Open the image property pane.
-* Click the **JS** button next to the **`onClick`** event and write the following JavaScript query:
-
-```
-{{download(atob(Fetch_image.data.fileData),'Testname','image/png')}}
-```
-
-Now, your image is downloaded when you click on the image widget.
-
-:::info
-You can check this [Guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more about Upload/Download Files from S3.
-:::
-
-With Appsmith S3 integration, it is possible to create apps that seamlessly connect with the S3 database and provide additional flexibility for updating and analyzing data.
-
+# Upload Images Using Camera
+Appsmith lets you capture images and videos from your applications and share the data for further use.
+For more information, see [Camera](https://docs.appsmith.com/reference/widgets/camera).
+
+This page provides steps to upload images using the Camera widget to a datasource and uses the S3 datasource as an example.
+You will learn how to:
+- [Create an S3 datasource and connect your app.](#create-an-s3-datasource)
+- [Upload an image to S3 using the Camera widget.](#upload-an-image-using-the-camera-widget)
+
+## Prerequisites
+- To upload your data to Amazon S3, you must first [create an Amazon S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) in one of the AWS Regions.
+
+## Create an S3 Datasource
```

Comment:
S3 datasource should be a part of the prerequisites, so this section is not needed. You could add that as a bullet after the Amazon S3 bucket to the prerequisites section:
- An Amazon S3 bucket. Link Amazon S3 bucket to official Amazon documentation for creating a bucket.
- A S3 datasource. For more information, see [how to create S3 datasource](Link to S3 reference doc).

Reply:
Done.